sys/net/gnrc/application_layer/uhcpc/gnrc_uhcpc.c

```diff
@@ -87,6 +90,14 @@ void uhcp_handle_prefix(uint8_t *prefix, uint8_t prefix_len, uint16_t lifetime,
 #if defined(MODULE_GNRC_IPV6_NIB) && GNRC_IPV6_NIB_CONF_6LBR && \
     GNRC_IPV6_NIB_CONF_MULTIHOP_P6C
     gnrc_ipv6_nib_abr_add((ipv6_addr_t *)prefix);
+#endif
+#ifdef MODULE_GNRC_RPL
+    gnrc_rpl_init(gnrc_wireless_interface);
+    gnrc_rpl_instance_t *inst = gnrc_rpl_instance_get(gnrc_wireless_interface);
+    if (inst) {
+        gnrc_rpl_instance_remove(inst);
+    }
+    gnrc_rpl_root_init(gnrc_wireless_interface, (ipv6_addr_t*)prefix, false, false);
```

Comment:
Hm, I don't like the idea of using `gnrc_wireless_interface` as the instance id. Instance IDs can be global or local, depening on the range (global for 0 <= x <= 127). In the unlikely event, that `gnrc_wireless_interface` is a very high number, this would lead to semantic side effects of initializing a local instance instead of a global one. I guess a global instance makes the most sense here.

I'd propose:
```
gnrc_rpl_root_init(0, (ipv6_addr_t*)prefix, true, false);
```

Reply:
Whoops, I misinterpreted some arguments there. @cgundogan How about reusing `GNRC_RPL_DEFAULT_INSTANCE` here and on line 96?

Reply:
Yeah, using `GNRC_RPL_DEFAULT_INSTANCE` is probably also nice (: 👍

Reply:
Fixed